build.sbt

```diff
@@ -32,7 +32,18 @@ lazy val mimaSettings = mimaDefaultSettings ++ Seq(
   mimaBinaryIssueFilters ++= Seq(
     ProblemFilters.exclude[DirectMissingMethodProblem]("play.libs.ws.ahc.StandaloneAhcWSResponse.getBodyAsSource"),
     ProblemFilters.exclude[MissingClassProblem]("play.api.libs.ws.package$"),
-    ProblemFilters.exclude[MissingClassProblem]("play.api.libs.ws.package")
+    ProblemFilters.exclude[MissingClassProblem]("play.api.libs.ws.package"),
+    ProblemFilters.exclude[IncompatibleResultTypeProblem]("play.libs.ws.ahc.*"),
```

Comment:
Hum... were those the filters suggested by Mima? I don't think that using wildcards here is a good idea because this will also inadvertently filter other changes in the future.

Better to list each problem individually.

Reply:
Just saw that mima was reporting the problems individually:

https://travis-ci.org/playframework/play-ws/jobs/404810938#L637

Better to use the filters suggested there.